beater/api/config/agent/handler.go

```diff
@@ -61,6 +63,122 @@ var (
 	rumAgents = []string{"rum-js", "js-base"}
 )
 
+// DirectConfigurationHandler returns a request.Handler for replying to agent
+// central configuration requests. The handler parses the incoming request and
+// responds with the config matching the service.name/service.environment pair.
+//
+// TODO: What if no matching service.name/service.environment pair is found?
```

Comment:
I'm assuming we want to return some form of error, and the agent will continue to run with defaults?